frontend/src/stores/settings.ts

```diff
@@ -0,0 +1,25 @@
+import { defineStore } from 'pinia';
+
+import { Settings, WalletType } from '@/types/settings';
```

Comment:
I don't think need to define `Settings` externally here. There is only this store who cares about the type anyway. All sub-types that are used within the store might be interesting. But be aware that type inheritance also works out of the box.